tests/meta/clean/conftest.py

```diff
@@ -1,59 +1,207 @@
+"""Tests for the CleanFlowCellsAPI."""
+from datetime import datetime
 from pathlib import Path
+from typing import Dict, List, Union
 
 import pytest
 
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import SequencingFileTag
+from cg.meta.clean.clean_flow_cells import CleanFlowCellAPI
+from cg.models.demultiplex.flow_cell import FlowCellDirectoryData
+from cg.store import Store
+from cg.store.models import Flowcell, Sample
+from tests.store_helpers import StoreHelpers
+
 
 @pytest.fixture(scope="function")
-def correct_flow_cell_name(bcl2fastq_flow_cell_full_name: str) -> str:
-    """Correct flow cell name."""
-    return bcl2fastq_flow_cell_full_name
+def flow_cell_clean_api_can_be_removed(
```

Comment:
The name is not clear
```suggestion
def clean_api_with_removable_flow_cell(
```

Reply:
There is another "cleanapi" which not related to flow cells which might be confusing, hence the flow_cell first approach